chart/templates/arkime_configmaps.yml

```diff
@@ -11,6 +11,11 @@ data:
   # The number of Arkime processes for analyzing uploaded PCAP files allowed
   #   to run concurrently
   ARKIME_AUTO_ANALYZE_PCAP_THREADS: "1"
+  # These variables control whether the Wise GUI is exposed as an additional endpoint.
+  # Controls whether the WISE configuration GUI is exposed via the nginx proxy
+  ARKIME_EXPOSE_WISE_GUI: "true"
+  # Controls whether the WISE configuration GUI is read-only or editable
+  ARKIME_ALLOW_WISE_GUI_CONFIG: "false"
```

Comment:
as the non-live arkime container is the only one that actually runs the wise process, this isn't needed in arkime-env, only in arkime-offline-env